framework/Source/MovieInput.swift

```diff
@@ -175,7 +175,7 @@ public class MovieInput: ImageSource {
         
         if let concreteLuminanceTextureRef = luminanceTextureRef, let concreteChrominanceTextureRef = chrominanceTextureRef,
             let luminanceTexture = CVMetalTextureGetTexture(concreteLuminanceTextureRef), let chrominanceTexture = CVMetalTextureGetTexture(concreteChrominanceTextureRef) {
-            let outputTexture = Texture(device:sharedMetalRenderingDevice.device, orientation:.portrait, width:bufferWidth, height:bufferHeight, timingStyle:.videoFrame(timestamp:Timestamp(withSampleTime)))
+            let outputTexture = Texture(device:sharedMetalRenderingDevice.device, orientation:.portrait, width:bufferWidth, height:bufferHeight, timingStyle:.videoFrame(timestamp:Timestamp(withSampleTime)))!
```

Comment:
same question for force unwrap

Reply:
The same for this
https://github.com/cardinalblue/GPUImage3/pull/22/files/6bdf51c39ee33945e1483c04aec6989c038885ed#r1645283263